build-azure.ps1

```diff
@@ -14,7 +14,7 @@ if (-not (Test-AppExists "vagrant")) {
 }
 Write-Output "Vagrant installed - good."
 
-if (-not (Test-AppExists "azure")) {
+if (-not (Test-AppExists "az")) {
```

Comment:
I tried to change some other stuff to work with `az` instead of `azure` - be careful, as lots is different...

Reply:
I never got the azure test to run, can remove this change